tests/test_models.py

```diff
@@ -56,3 +57,20 @@ def test_delete_a_promotion(self):
         promotion.delete()
         self.assertEqual(len(Promotion.all()), 0)
 
+    def test_update_no_id_should_raise_error(self):
+        promo = PromotionsFactory()
```

Comment:
```suggestion
        promotion = PromotionsFactory()
```
The previous code is following `promotion` convention